Remotion/Web/Development.WebTesting/Accessibility/Implementation/AccessibilityResultMapper.cs

```diff
@@ -128,11 +135,17 @@ private IEnumerable<AccessibilityTestSuccessCriteria> GetSuccessCriteria (string
       return ruleTags
           .Where(t => !AccessibilityConformanceLevelConverter.IsValid(t))
           .Where(t => !_wellKnownSuccessCriteriaToIgnore.Contains(t))
+          .Where(t => !_wellKnownSuccessCriteraPrefixesToIgnore.Any(t.StartsWith))
```

Comment:
Using a hash set here does not make any sense if we iterate over it anyway

Reply:
Those were my exact thoughts when looking at that last week 😅